usr/share/cachyos-hyprland-settings/decorations.conf

```diff
@@ -8,22 +8,13 @@ source = /usr/share/cachyos-hyprland-settings/colors.conf
 
 decoration {
     active_opacity = 0.98
-    inactive_opacity = 1.0
-    fullscreen_opacity = 1.0
     rounding = 4
-    dim_inactive = false
-    #dim_special = 0.2
-    #dim_around = 0.4
-    #screen_shader =
-    
+
     # https://wiki.hyprland.org/0.45.0/Configuring/Variables/#blur
     blur {
-        enabled = true
         size = 15
         passes = 2 # more passes = more resource intensive.
-        new_optimizations = true
```

Comment:
You should leave this one but having the blur itself disabled. no need to remove it

Reply:
`true` is the default, any reason to keep it other than for the sake of posterity?

Reply:
if its the default then sure, go ahead, i'm not to up to date on what's currently on by default in Hyprland